wrappers/matlab/option.m

```diff
@@ -83,6 +83,8 @@
         sequence_id                     (79)
         humidity_temperature            (80)
         max_usable_range                (81)
-        count                           (82)
+        noise_estimation                (82)
```

Comment:
You'll get conflicts with all of there because of Alternate IR...

Reply:
👍